error: variable length arrays in C++ are a Clang extension [-Werror,-Wvla-cxx-extension]

Build fails with clang-18:
```
/wrkdirs/usr/ports/multimedia/vvenc/work/vvenc-1.11.1/source/Lib/EncoderLib/InterSearch.cpp:2064:15: error: variable length arrays in C++ are a Clang extension [-Werror,-Wvla-cxx-extension]
 2064 |     Mv prevMv[m_BlkUniMvInfoBuffer->m_uniMvListMaxSize];
      |               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/wrkdirs/usr/ports/multimedia/vvenc/work/vvenc-1.11.1/source/Lib/EncoderLib/InterSearch.cpp:2064:15: note: implicit use of 'this' pointer is only allowed within the evaluation of a call to a 'constexpr' member function
1 error generated.
```

[log](https://pkg-status.freebsd.org/beefy18/data/main-amd64-default/pb7573d3199cc_sfb8a8333b48/logs/vvenc-1.11.1.log)

Version: 1.11.1
FreeBSD 15-CURRENT

--- a/source/Lib/EncoderLib/InterSearch.cpp
+++ b/source/Lib/EncoderLib/InterSearch.cpp
@@ -2061,7 +2061,7 @@ void InterSearch::xMotionEstimation(CodingUnit& cu, CPelUnitBuf& origBuf, RefPic
     Distortion uiBestSad = m_cDistParam.distFunc(m_cDistParam);
     uiBestSad += m_pcRdCost->getCostOfVectorWithPredictor(cTmpMv.hor, cTmpMv.ver, cStruct.imvShift);
 
-    Mv prevMv[m_BlkUniMvInfoBuffer->m_uniMvListMaxSize];
+    Mv prevMv[BlkUniMvInfoBuffer::m_uniMvListMaxSize];
 
     for( int i = 0; i < m_BlkUniMvInfoBuffer->m_uniMvListSize; i++ )
     {

--- a/source/Lib/EncoderLib/InterSearch.h
+++ b/source/Lib/EncoderLib/InterSearch.h
@@ -193,10 +193,10 @@ struct BlkUniMvInfoBuffer
     m_uniMvListSize = 0;
   }
 
-  static const int m_uniMvListMaxSize = 15;
-  BlkUniMvInfo     m_uniMvList[m_uniMvListMaxSize];
-  int              m_uniMvListIdx;
-  int              m_uniMvListSize;
+  static constexpr int m_uniMvListMaxSize = 15;
+  BlkUniMvInfo         m_uniMvList[m_uniMvListMaxSize];
+  int                  m_uniMvListIdx;
+  int                  m_uniMvListSize;
 };
 
 class EncPicture;

